.github/workflows/test.yml

```
@@ -13,10 +13,29 @@ jobs:
         run: bash tests/test_list_prs_sort.sh
+
+  aw-compile:
+    runs-on: ubuntu-latest
+    timeout-minutes: 3
+    steps:
+      - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd  # v6.0.2
+        with:
+          persist-credentials: false
+
+      - name: Install Python dependencies
+        run: pip install "pyyaml==6.0.3" --only-binary :all: --quiet
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)